css/bodystyle.css

```diff
@@ -0,0 +1,200 @@
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+}
+
+nav {
+  background-color: white;
+  height: 60px;
+  width: 100%;
+}
+
+img.logo {
+  width: 20px;
+  height: auto;
+  margin: 10px;
+  margin-left: 40px;
+}
+
+nav ul {
+  float: right;
+  margin: 0 20px;
+}
+
+nav ul li {
+  display: inline;
+  line-height: 60px;
+  margin: 0 5px;
+}
+
+nav ul li a {
+  color: rgb(68, 67, 67);
+  font-size: 20px;
+  padding: 5px 10px;
+}
+
+a:hover {
+  color: rgb(197, 58, 58);
+  transition: 0.5s;
+}
+
+.menubutton {
+  float: right;
+  color: rgb(68, 67, 67);
+  font-size: 20px;
+  line-height: 60px;
+  margin-right: 40px;
+  cursor: pointer;
+  display: none;
+}
+
+#check {
+  display: none;
+}
+
+main {
+  width: 100%;
+}
+/* HERO */
+#hero div {
+  min-height: 400px;
+  background: url("../img/first-background.jpg") no-repeat;
+  background-size: cover;
+  background-position: right;
+  color: whitesmoke;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: auto;
+  text-align: center;
+  padding: 20%;
+}
+
+#hero h2 {
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
+}
+
+#hero p {
+  font-size: 15px;
+  margin: 10px;
+}
+
+#hero button {
+  margin-top: 20px;
+  width: 120px;
+  height: 40px;
+  font-size: 14px;
+  background-color: rgb(225, 106, 66);
```

Comment:
Can you add a border-radius on the button and remove the default border to match the design?